### ISSUE TITLE
Numbering

### ISSUE THREAD

**Issue by andrewjskatz** [OpenChain-Project/License-Compliance-Specification#39]
The numbering seems inconsistent: the numbering for the verification materials seems disconnected from the rest of the numbering. For example, the Verification Materials for 4.1.1 have the numbers 1.1.1 and 1.1.2; the Verification Materials for 4.1.2 Competence have 1.2.1, 1.2.2 and 1.2.3 and so on.

**Comment by MarkGisi** [comment]
Initially people wanted to ensure we maintained the requirement numbering for backward compatibility to the previous versions. The awkwardness you point out came about due to retrofitting the previous spec version into the current ISO formatting and numbering scheme. 

One compromise might be to add  the "4." to the requirement version. For example:
1.2.1, 1.2.2 and 1.2.3   --> 4.1.2.1, 4.1.2.2 and 4.1.2.3 . This would require the Self Certification web application to be update to accommodate the new version numbering. If we are going to make a change like this, this would be the time to do it (as opposed to a later version of the spec.)

I added it to the "first" section of the current draft - let's see what the community feedback is.
